package.json

```diff
@@ -13,21 +13,22 @@
     "errorhandler": "^1.2.1",
     "event-stream": "^3.1.7",
     "express": "^4.9.7",
-    "frozen-moment": "^0.1.1",
-    "lodash": "^2.4.1",
+    "frozen-moment": "^0.4.0",
+    "kerberos": "~0.0",
+    "lodash": "^4.17.4",
     "ludwig-api": "^1.2.0",
     "moment": "^2.8.3",
-    "mongoose": "~3.8.18",
+    "mongoose": "^4.8.3",
     "morgan": "^1.3.2",
-    "superagent": "^0.21.0",
-    "underscore.string": "^2.3.3"
+    "superagent": "^3.4.4",
```

Comment:
From https://github.com/visionmedia/superagent/blob/master/History.md#100--2015-03-08: this changes the error handling behaviour.

`lib/simulation/openfisca/index.js#L14` is indeed impacted.

Please either update that code or drop `superagent` altogether in favour of the native `https` module, since it is used only in this one place.

I did not check the impact of any other changes from `superagent`.